src/Car.java

```diff
@@ -1,14 +1,14 @@
 
 public class Car {
-	
+
 	private int id;
 	private String marka;
 	private String model;
 	private int productionYear;
 	private String color;
 	private int price;
 	private int registrationNumber;
-	
+
 	public Car(int id, String marka, String model, int productionYear, String color, int price, int registrationNumber) {
```

Comment:
If you are going to add something to the constructor in the future - you should think about creation additional class for primitive variables it would be more useful and conveniently.